### ISSUE TITLE
The application closes after about 36 hours

### ISSUE THREAD

**Issue by vsatyamesc** [noahc3/RedditDiscordRSSBot#2]
I have a RDP that I use to run this bot, but this application closes after 36 hours or so, I don't know the exact time, but it closes after 30 hours guaranteed. can you do something so that the application doesn't closes? I have to manually turn this on everytime and it is really problematic.
I use Windows 10 64bit

**Comment by noahc3** [comment]
Yes, there are a number of crashing issues when the reddit RSS API returns malformed responses. On my end I'm currently running it through a bash script that relaunches it when it crashes but I know that's not ideal, I'll look into better error handling at some point.

**Comment by vsatyamesc** [comment]
If possible can you send the script?

**Comment by noahc3** [comment]
The one I use is for Linux (bash):
```
while true
do
        ./RedditDiscordRSSBot-linux64
        sleep 1
done
```

Here's one that should work on Windows as a .bat file, but I haven't tested it:
```
:loop
START /WAIT RedditDiscordRSSBot-win64.exe
timeout 1
goto loop
```
